frontend/apps/vara-man/src/app/hocs/index.tsx

```diff
@@ -1,29 +1,77 @@
-import type { ComponentType } from 'react';
-import { BrowserRouter } from 'react-router-dom';
+import type { ComponentType } from 'react'
+import { BrowserRouter } from 'react-router-dom'
 import {
-  ApiProvider as GearApiProvider,
-  AlertProvider as GearAlertProvider,
-  AccountProvider,
-  ProviderProps,
-} from '@gear-js/react-hooks';
-import { ENV } from '@/app/consts';
-import { AppProvider } from '@/app/context/ctx-app';
-import { GameProvider } from '@/app/context/ctx-game';
-import { Alert, alertStyles } from '@/components/ui/alert';
+	ApiProvider as GearApiProvider,
+	AlertProvider as GearAlertProvider,
+	AccountProvider,
+	ProviderProps,
+} from '@gear-js/react-hooks'
+import {
+	SignlessTransactionsProvider as SharedSignlessTransactionsProvider,
+	GaslessTransactionsProvider as SharedGaslessTransactionsProvider,
+	EzTransactionsProvider
+} from '@dapps-frontend/ez-transactions'
+import { ENV } from '@/app/consts'
+import { AppProvider } from '@/app/context/ctx-app'
+import { GameProvider } from '@/app/context/ctx-game'
+import { Alert, alertStyles } from '@/components/ui/alert'
+import metaTxt from '@/assets/meta/vara_man.meta.txt'
 
 const ApiProvider = ({ children }: ProviderProps) => (
-  <GearApiProvider initialArgs={{ endpoint: ENV.NODE }}>{children}</GearApiProvider>
-);
```

Comment:
Most changes are just formatting.
Have you check lint config?

Reply:
Yeah, my config is a little off, I'll fix it.